lib/active_model/serializer.rb

```diff
@@ -162,7 +180,7 @@ def self.serializers_cache
 
     def self.get_serializer_for(klass)
       serializers_cache.fetch_or_store(klass) do
-        serializer_class_name = "#{klass.name}Serializer"
+        serializer_class_name = "#{klass.name}Serialization"
         serializer_class = serializer_class_name.safe_constantize
```

Comment:
@joaomdmoura how about we move the renaming commit into its own PR for clarity? :)

```diff
-        serializer_class_name = "#{klass.name}Serializer"
+        serializer_class_name = "#{klass.name}Serialization"
```